### PR TITLE
Adding suport to stop build api

### PR DESCRIPTION
**Pull request by mateusfreira**

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -78,6 +79,19 @@ var init = exports.init = function(host, options) {
...
+        stopBuild: function(jobname, buildNumber, callback) {
+            var buildurl;
+            buildurl = build_url(STOP_BUILD, jobname, buildNumber);
+
+            request({method: 'POST', url: buildurl }, function(error, response) {
+                if ( error || (response.statusCode !== 201 && response.statusCode !== 302) ) {
+                    callback(error, response);
+                    return;
+                }
+                var data = "job is stoped";
```
Comment (jansepar) [inline lib/main.js]: stopped*

**Comment by jansepar** [comment]
Would be great to add this command to the README!

**Comment by mateusfreira** [comment]
@jansepar 
Thanks for your considerations. was fixed in the next commits.

File: lib/main.js [hunk 2]
```diff
@@ -78,6 +79,19 @@ var init = exports.init = function(host, options) {
                 callback(null, data);
             });
         },
+        stopBuild: function(jobname, buildNumber, callback) {
```
Comment (brianmichel) [inline lib/main.js]: Could this match the format of other commands and variable using the `_` to denote spaces? So `stopBuild` → `stop_build`? Obviously up to @jansepar, but it looks weird to mix formats to me.

Comment (mateusfreira) [inline lib/main.js]: @brianmichel  bd015e0 fix this. thank you.

**Comment by brianmichel** [comment]
Hey @jansepar what's going on with this PR, can it be merged?

**Comment by jansepar** [comment]
Hi guys, sorry for the delay on this, it's looking good thanks! :+1:

**Pull request merged**